PopApp/websockets/LocalMockServer.js

```diff
@@ -3,6 +3,8 @@ const webSocketsServerPort = 8000;
 const webSocketServer = require('websocket').server;
 const http = require('http');
 
+// FIXME: Could this be re-used further ? If not, remove it entirely.
+
```

Comment:
Probably only useful for low-level websocket testing. It was also supposed to be used in unit tests, but GitHub actions have problems running the server

Reply:
With Gaurav we were envisioning the possibility of having something like this, both to test the backends and the frontends.

We'll have to think a bit more about it though.

Are you sure you didn't have a problem like this?
https://github.community/t/test-http-server-in-github-actions/17608/6